### DISCUSSION TITLE
多配置Webhook是否会重复通知？

### DISCUSSION THREAD

**Discussion by qhq** [jeessy2/ddns-go#842] [Q&A]
弄了两个配置，Webhook好像是通用的，就和登录账号密码一样，两个配置里都会显示登录账号密码和Webhook，删一个就全删。保存的话，更新ip的以后就会因为两个配置通知两次。
如果是的话是否可以考虑优化一下。

**Comment by jeessy2** [comment]
有几个配置就会有多少个通知，设计就是这样的
